src/docs/zksync-era.mdx

```diff
@@ -115,8 +115,6 @@ links:
     | 38 | CODESIZE | | Gets the size of the code in case it is called on a Runtime code.<br /><br />If called on a deploy code, the returned size is the size of the constructor arguments only (excluding deploy code size). | Gets the size of the code running in the current environment. <Modified /> |
     | 39 | CODECOPY | | The Rollup does not provide a mechanism to access the bytecode of a contract. <br /><br />Copies only constructor arguments when executed on deploy code.<br /><br />Sets memory to `0`s when executed on runtime code compiled through old EVM codegen.<br /><br />Produces compile-time error in case it is executed on runtime code compiling through Yul codegen. | Copies the code running in the current environment to memory. |
     | 41 | COINBASE | | Returns the address of the `Bootloader` contract, which is `0x8001` | Gets the block’s beneficiary address <Modified />  |
-    | 42 | TIMESTAMP | | Returns the latest L1 batch timestamp | Returns the latest block’s timestamp <Modified /> |
-    | 43 | NUMBER | | Returns the latest L1 batch number | Returns the latest block’s number <Modified /> |
```

Comment:
Rather than removing the "change" we need to specify that those opcodes return the L2 block number and timestamp. For the rest of the rollups we are doing the same you can see how we are specifying it there